### PR TITLE
WIP Add utility script to dev godocs

### PR DESCRIPTION
**Pull request by izgeri**
- Add script to facilitate viewing godocs locally
- Update CONTRIBUTING.md with info on documentation sources / doc dev
- Move docs/README content into CONTRIBUTING

#### What ticket does this PR close?
Connected to #642 

#### Where should the reviewer start?
To test this PR, you can check out the branch and run `./bin/dev_godocs`. Visit `localhost:6060` in your browser to view the godocs for the project, and make a change to a `.go` file and see it reflected in the browser.

#### What is the status of the manual tests?
Have you run the following manual tests to verify existing functionality continues to function as expected?
- [ ] Manually tested [K8s CRDs](https://github.com/cyberark/secretless-broker/tree/master/test/manual/k8s_crds)
- [ ] Manually tested [Keychain provider](https://github.com/cyberark/secretless-broker/tree/master/test/manual/keychain_provider)
- [ ] Manually run the [K8s demo](https://github.com/cyberark/secretless-broker/tree/master/demos/k8s-demo)
- [ ] Manually run [Kubernetes-Conjur demo](https://github.com/conjurdemos/kubernetes-conjur-demo) with a local Secretless Broker image build of your branch
- [ ] Manually run the [full demo](https://github.com/cyberark/secretless-broker/tree/master/demos/full-demo) (optional)

If this feature does not have any/sufficent automated tests, have you created/updated a folder in `test/manual` that includes:
- [ ] An updated README with instructions on how to manually test this feature
- [ ] Utility `start` and `stop` scripts to spin up and tear down the test environments
- [ ] A `test` script to run some basic manual tests (optional; if does not exist, the README should have detailed instructions)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -134,6 +135,40 @@ cd test/manual/k8s_crds
...
+### Website
+The [website](https://secretless.io) source is in the [docs](docs/) folder in this repository. It is generated using Jekyll.
+
+#### Prerequisites
+
+To get the site up and running locally on your computer, ensure you have:
+1. Ruby version 2.1.0 or higher (check by running `ruby -v`)
+2. Bundler (`gem install bundler`)
+3. Jekyll (`gem install jekyll`)
+4. Once Bundler and Jekyll gems are installed, run `bundle install`
```
Comment (jonahx) [inline CONTRIBUTING.md]: Shouldn't you only have to run `bundle install` if you've added the deps to your Gemfile?  

But if we're locally running `gem install bundler` and `gem install jekyll` you shouldn't also need to run `bundle install`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -134,6 +135,40 @@ cd test/manual/k8s_crds
...
+3. Run the following command:
+`bundle exec jekyll serve`
+4. Preview Jekyll site locally in web browser by either running `open localhost:4000` or manually navigating to http://localhost:4000
+
+#### Run in Docker
+With `docker` and `docker-compose`:
+
+1. Run `docker-compose up -d` in the `docs` directory.
+2. Preview Jekyll site locally in web browser by either running `open localhost:4000` or manually navigating to http://localhost:4000
+
```
Comment (jonahx) [inline CONTRIBUTING.md]: Let's note that with both method, changes will be picked up without a server restart.

File: CONTRIBUTING.md (hunk 1 above)
Comment (izgeri) [reply CONTRIBUTING.md]: I dunno - I just copied the instructions that were in the docs/README, and they've never let me down :)

**Comment by izgeri** [comment]
Doc updates are in #678, but the godoc updates aren't working very well right now due to the complexity of being a multi-package project. Since it'll likely require a more complex solution, closing this PR for now.

**Pull request closed without merging**